exchanges/order/limits_test.go

```diff
@@ -161,45 +140,31 @@ func TestGetOrderExecutionLimits(t *testing.T) {
 	}
 
 	err = e.LoadLimits(newLimits)
-	if !errors.Is(err, nil) {
-		t.Fatalf("expected error %v but received %v", errCannotLoadLimit, err)
-	}
+	require.NoError(t, err)
 
 	_, err = e.GetOrderExecutionLimits(asset.Futures, ltcusd)
-	if !errors.Is(err, ErrCannotValidateAsset) {
-		t.Fatalf("expected error %v but received %v", ErrCannotValidateAsset, err)
-	}
+	require.ErrorIs(t, err, ErrCannotValidateAsset)
 
 	_, err = e.GetOrderExecutionLimits(asset.Spot, ltcusd)
-	if !errors.Is(err, errExchangeLimitBase) {
-		t.Fatalf("expected error %v but received %v", errExchangeLimitBase, err)
-	}
+	require.ErrorIs(t, err, errExchangeLimitBase)
 
 	_, err = e.GetOrderExecutionLimits(asset.Spot, btcltc)
-	if !errors.Is(err, errExchangeLimitQuote) {
-		t.Fatalf("expected error %v but received %v", errExchangeLimitQuote, err)
-	}
+	require.ErrorIs(t, err, errExchangeLimitQuote)
 
 	tt, err := e.GetOrderExecutionLimits(asset.Spot, btcusd)
-	if !errors.Is(err, nil) {
-		t.Fatalf("expected error %v but received %v", nil, err)
-	}
+	require.NoError(t, err)
 
-	if tt.MaximumBaseAmount != newLimits[0].MaximumBaseAmount ||
+	assert.False(t, tt.MaximumBaseAmount != newLimits[0].MaximumBaseAmount ||
 		tt.MinimumBaseAmount != newLimits[0].MinimumBaseAmount ||
 		tt.MaxPrice != newLimits[0].MaxPrice ||
-		tt.MinPrice != newLimits[0].MinPrice {
-		t.Fatal("unexpected values")
-	}
+		tt.MinPrice != newLimits[0].MinPrice)
```

Comment:
Please rewrite this into more readable code. Each of these can be an individual `assert` against the value